src/current/v24.3/logical-data-replication-overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Logical Data Replication
+summary: An overview of CockroachDB logical data replication (LDR).
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v24.3" %} **Logical data replication (LDR)** continuously replicates tables between active CockroachDB clusters. Both source and destination cluster can receive application reads and writes, with LDR enabling bidirectional replication for eventual consistency in the replicating tables. The active-active setup can provide protection against cluster, datacenter, or region failure while still achieving single-region low latency reads and writes in the individual CockroachDB clusters. Each cluster in an LDR job still benefits individually from [multi-active availability]({% link {{ page.version.version }}/multi-active-availability.md %}) with CockroachDB's built-in [Raft replication]({% link {{ page.version.version }}/demo-replication-and-rebalancing.md %}) providing data consistency across nodes, zones, and regions.
+
+{{site.data.alerts.callout_success}}
+Cockroach Labs also has a [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) tool that continuously sends data at the byte level for transactional consistency from a primary cluster to an independent standby cluster.
+{{site.data.alerts.end}}
+
+## Use cases
+
+You can run LDR in a _unidirectional_ or _bidirectional_ setup to meet different use cases that support:
+
+- [High availability and single-region write latency in two-datacenter deployments](#achieve-high-availability-and-single-region-write-latency-in-two-datacenter-deployments)
+- [Workload isolation between clusters](#achieve-workload-isolation-between-clusters)
+
+### Achieve high availability and single-region write latency in two-datacenter deployments
+
+Maintain [high availability]({% link {{ page.version.version }}/data-resilience.md %}#high-availability) with a two-datacenter topology. You can run bidirectional LDR to ensure [data resilience]({% link {{ page.version.version }}/data-resilience.md %}) in your deployment, particularly in datacenter or region failures. Both clusters can receive application reads and writes with low, single-region write latency. In a datacenter or cluster outage, you can redirect application traffic to the surviving cluster with [low downtime]({% link {{ page.version.version }}/data-resilience.md %}#high-availability). In the following diagram, the clusters are deployed in US East and West to provide low latency for that region. The two LDR jobs ensure that the tables on both clusters will reach eventual consistency.
```

Comment:
> Both clusters can receive application reads and writes with low, single-region write latency. 

can we start this with "If you set up two single-region clusters, in LDR, both clusters can receive..."

>  In a datacenter or cluster outage, you can redirect application traffic to the surviving cluster with 

"Then, in a datacenter, region, or cluster outage"?

>  In the following diagram, the clusters are deployed in US East and West to provide low latency for that region. 

the two single-region clusters?

Reply:
Done